[2023-09-04T06:01:56.210+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-02-20T00:00:00+00:00 [queued]>
[2023-09-04T06:01:56.220+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-02-20T00:00:00+00:00 [queued]>
[2023-09-04T06:01:56.221+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:01:56.241+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-02-20 00:00:00+00:00
[2023-09-04T06:01:56.248+0000] {standard_task_runner.py:57} INFO - Started process 5083 to run task
[2023-09-04T06:01:56.256+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-02-20T00:00:00+00:00', '--job-id', '329', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpb8m8kahd']
[2023-09-04T06:01:56.259+0000] {standard_task_runner.py:85} INFO - Job 329: Subtask insert_into_table
[2023-09-04T06:01:56.326+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-02-20T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:01:56.430+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-02-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-20T00:00:00+00:00'
[2023-09-04T06:01:56.432+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-02-20', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:01:56.442+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:01:56.451+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:01:56.465+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-02-20', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:01:56.469+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:01:56.486+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220220T000000, start_date=20230904T060156, end_date=20230904T060156
[2023-09-04T06:01:56.545+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:01:56.565+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:04:46.048+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-02-20T00:00:00+00:00 [queued]>
[2023-09-04T06:04:46.082+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-02-20T00:00:00+00:00 [queued]>
[2023-09-04T06:04:46.083+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:04:46.140+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-02-20 00:00:00+00:00
[2023-09-04T06:04:46.156+0000] {standard_task_runner.py:57} INFO - Started process 6411 to run task
[2023-09-04T06:04:46.166+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-02-20T00:00:00+00:00', '--job-id', '889', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpon_uo2e6']
[2023-09-04T06:04:46.171+0000] {standard_task_runner.py:85} INFO - Job 889: Subtask insert_into_table
[2023-09-04T06:04:46.346+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-02-20T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:04:46.643+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-02-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-20T00:00:00+00:00'
[2023-09-04T06:04:46.646+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-02-20', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:04:46.691+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:46.719+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:46.786+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-02-20', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:04:46.802+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:04:46.862+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220220T000000, start_date=20230904T060446, end_date=20230904T060446
[2023-09-04T06:04:46.950+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:04:46.995+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
